cmd/root.go

```diff
@@ -114,13 +113,16 @@ func rootCmdPersistentFlagSet() *pflag.FlagSet {
 }
 
 func init() {
-	// TODO: find a better library... or better yet, simply port the few dozen lines of code for getting the
-	// per-user config folder in a cross-platform way
-	configDirs := configdir.New("loadimpact", "k6")
-	configFolders := configDirs.QueryFolders(configdir.Global)
-	if len(configFolders) > 0 {
-		defaultConfigFilePath = filepath.Join(configFolders[0].Path, defaultConfigFileName)
+	confDir, err := configDir()
+	if err != nil {
+		panic(err)
```

Comment:
Can you do some testing with docker to see that we don't run into this?

Reply:
@MStoykov I did this in my local:

```
$ docker build -t cuonglm/k6 .
$ docker run -v "$PWD":/app --rm cuonglm/k6:latest run /app/samples/http_get.js
```

And it runs ok.